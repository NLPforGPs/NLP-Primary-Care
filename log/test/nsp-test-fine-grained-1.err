cpu-bind=MASK - bp1-gpu003, task 4294967295 4294967295 [0]: mask 0x1010 set
usage: conda-env [-h] {create,export,list,remove,update,config} ...
conda-env: error: invalid choice: 'createa' (choose from 'create', 'export', 'list', 'remove', 'update', 'config')
[nltk_data] Downloading package wordnet to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package wordnet is already up-to-date!
[nltk_data] Downloading package punkt to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
Some weights of the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract were not used when initializing BertForNextSentencePrediction: ['cls.predictions.transform.dense.weight', 'cls.predictions.decoder.weight', 'cls.predictions.bias', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.decoder.bias']
- This IS expected if you are initializing BertForNextSentencePrediction from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForNextSentencePrediction from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
2021-12-09 13:53:56,358 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:155] - INFO: Predicting...
2021-12-09 13:53:56,388 - /user/home/zu20361/.conda/envs/NLP_GP/lib/python3.9/site-packages/datasets/builder.py[line:377] - WARNING: Using custom data configuration default-a2d0abb6534a5099
0 examples [00:00, ? examples/s]2021-12-09 13:53:56,569 - /user/home/zu20361/.cache/huggingface/modules/datasets_modules/datasets/transcript_evaldataset/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330/transcript_evaldataset.py[line:59] - INFO: Generating examples...
                                  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00, 389.52it/s]
2021-12-09 13:53:56,671 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:193] - INFO: NSP method
  0%|          | 0/1 [00:00<?, ?ba/s]100%|██████████| 1/1 [00:31<00:00, 31.56s/ba]100%|██████████| 1/1 [00:31<00:00, 31.56s/ba]

real	5m27.479s
user	3m49.424s
sys	1m31.311s
