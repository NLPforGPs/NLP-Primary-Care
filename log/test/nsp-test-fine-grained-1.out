JOB ID: 264646
Working Directory: /user/home/zu20361/NLP-Primary-Care
Start Time: Thu Dec 9 13:53:40 GMT 2021
loading codes from PE...
Downloading and preparing dataset description_dataset/default (download: Unknown size, generated: 2.07 MiB, post-processed: Unknown size, total: 2.07 MiB) to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330...
Dataset description_dataset downloaded and prepared to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330. Subsequent calls will reuse this data.
classification_report:
                 precision    recall  f1-score   support

          blood       0.06      1.00      0.11         8
 cardiovascular       0.32      0.84      0.46        32
      digestive       0.39      0.84      0.53        44
            ear       0.38      1.00      0.55        11
      endocrine       0.23      0.58      0.33        24
            eye       0.09      1.00      0.17         5
         female       0.16      0.79      0.27        14
        general       0.06      1.00      0.11        14
           male       0.07      0.71      0.13         7
musculoskeletal       0.70      0.86      0.77        65
   neurological       0.23      0.75      0.35        20
      pregnancy       0.26      0.64      0.37        11
  psychological       0.21      1.00      0.35        50
    respiratory       0.52      0.86      0.65        37
           skin       0.23      0.88      0.36        32
     urological       0.43      1.00      0.60        18

      micro avg       0.23      0.86      0.36       392
      macro avg       0.27      0.86      0.38       392
   weighted avg       0.36      0.86      0.47       392
    samples avg       0.25      0.87      0.37       392

f1_score 0.38190084665301416
roc_auc 0.8651570959646999
End Time: Thu Dec 9 13:59:07 GMT 2021
